PhotoPoints/app/src/main/java/edu/cascadia/mobas/photopoints/model/PhotoPoint.java

```diff
@@ -14,27 +14,27 @@
     private Double mLongitude;
     private PhotoPointType mPhotoPointType;
     private String mQRCode;
+    private int mItemID;      // ID of item (plant, creek) this PhotoPoint refers to
 
-    public PhotoPoint(int photoPointID, Double latitude, Double longitude){
+
+    // provided for compatibility with existing data model
+    public PhotoPoint(int photoPointID, Double latitude, Double longitude, String qrCode, int photoPointType) {
+        this(photoPointID, latitude, longitude, qrCode, photoPointType, 1);
```

Comment:
Is it a type that we hard-code the item ID here?

Reply:
A temporary patch, which should be removed, along with the whole constructor..  The other constructor has an additional constructor for the ItemID.  No reason to use case to instantiate with everything but this crucial field.

Reply:
WIll be removed in the future - for now ensures that no conflicts exist